lennonFolder/lennon-github-serach/simple-github/src/main/kotlin/com/androidhuman/example/simplegithub/ui/search/SearchAdapter.kt

```diff
@@ -0,0 +1,82 @@
+package com.androidhuman.example.simplegithub.ui.search
+
+import android.graphics.Color
+import android.graphics.drawable.ColorDrawable
+import android.support.v7.widget.RecyclerView
+import android.text.TextUtils
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import com.androidhuman.example.simplegithub.R
+import com.androidhuman.example.simplegithub.api.model.GithubRepo
+import com.androidhuman.example.simplegithub.ui.GlideApp
+import java.util.*
+
+/**
+ * p.245 코틀린 익스텐션은 구글이 버리는것 같아서 적용하지 않았음. 대신 뷰 바인딩을 씁시다.
+ */
+class SearchAdapter : RecyclerView.Adapter<SearchAdapter.RepositoryHolder>() {
+    private var items: MutableList<GithubRepo> = mutableListOf()
+    private val placeholder = ColorDrawable(Color.GRAY)
+    private var listener: ItemClickListener? = null
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): RepositoryHolder =
```

Comment:
보통 뷰홀더에 parent를 보다 여기서 레이아웃을 inflate 해서 넘겨줍니다~

```suggestion 
override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): RepositoryHolder {
    val view = LayoutInflater.from(parent.context).inflate(R.layout.item_repository, parent, false)
    return RepositoryHolder(view)
}
```

Reply:
뷰바인딩을 사용한다면 바인딩을 넘겨줍니다~

```
override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): RepositoryHolder {
   val binding = ItemRepositoryBinding.inflate(
            LayoutInflater.from(parent.context),
            parent,
            false
)
    return RepositoryHolder(binding)
}

```

Reply:
뷰바인딩 사용해보겠습니다.